backend/routes/db.py

```diff
@@ -343,16 +275,164 @@ async def get_concepts_post_route(request: Request, id: Union[List[str], None] =
     return await get_concepts_route(request, id=id, table=table)
 
 
+sessions_cache: Dict = {}
+session_stale_hours = 24
+
+
+@router.get("/start-session")
+def start_session():
+    return get_session(None)
+
+
+def get_session(session_id: int):
```

Comment:
```suggestion
def get_session(session_id: Union[int, None]):
```

or

```suggestion
def get_session(session_id: Union = None):
```
and change `return get_session(None)` to `return get_session()`